pkg/controller/import-controller.go

```diff
@@ -99,19 +102,30 @@ func (ic *ImportController) processPvcItem(pvc *v1.PersistentVolumeClaim) error
 	if pod == nil && needsSync {
 		var podEnvVar importPodEnvVar
 
+		//check for endpoint annotation
 		podEnvVar.ep, err = getEndpoint(pvc)
```

Comment:
Nat a changed in this PR, but it seems like is is OK for a pvc to not have an endpoint.  So maybe that case shouldn't be treated as an error?  

Maybe it would be best to have a function that returns a fully populated `importPodEnvVar` based on a PVC, then the code can make decisions based on that?

Reply:
I'm not sure I understood.
"So maybe that case shouldn't be treated as an error?"
In my flow, if we do not find endpoint anno,we look for blankImage anno. If both does not exist, it's an error. Does not it make sense?

Reply:
Sure, but on line 108, an error is logged if PVC does not have an endpoint.  That is not really an error as it is expected in the blank disk use case.

I personally think it may be cleaner to get rid of `getEndpoint()`, `getBlankInage()`, and `getSectetName()` (and any other similar functions that assign members of podEnvVar) and have a single function that takes a PVC and returns a fully initialized `importPodEnvVar`.  The controller can then make decisions (is this an import or a blank disk) based on the info in the structure.

Reply:
Oh.. I see now. Sure. I will write one method to do that...